pkg/resources/resource_monitor.go

```diff
@@ -335,17 +335,13 @@ func UpdateResourceMonitor(d *schema.ResourceData, meta interface{}) error {
 		opts.Set.CreditQuota = sdk.Pointer(d.Get("credit_quota").(int))
 	}
 
-	if d.HasChange("frequency") {
+	if d.HasChange("frequency") || d.HasChange("start_timestamp") {
```

Comment:
There is only one acceptance test for resource monitor that is currently run. It would be nice to have one showing the issue you are fixing (should fail before the fix, but should be okay after). Can you add it?

Reply:
Hey, yes I'll add 👍

Reply:
Hey, I have added a acceptance test for start and endtime stamps